src/components/Cart/Steps/Steps.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="np-steps">
+    <div v-for="(stepName, index) in $tm('menuSteps',{returnObjects: true})"
+         :key="index"
+         class="np-steps__wrapper">
+
+      <div :class="{
```

Comment:
if theres more complicated logic then simple comparison, then move it to methods (setup) and just call function here
somsething like calculateClass(index)